.github/workflows/srsim-tests.yml

```diff
@@ -56,10 +66,20 @@ jobs:
       - name: Sanitize test-suite name
         run: echo "TEST_SUITE=$(echo ${{ matrix.test-suite }} | tr -d '*')" >> $GITHUB_ENV
 
-      - name: Run sros tests
-        run: |
-          bash ./tests/rf-run.sh ${{ matrix.runtime }} ./tests/13-srsim/${{ matrix.test-suite }}
+      - name: Login to GitHub Container Registry
+        uses: docker/login-action@v3
+        with:
+          registry: ghcr.io
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+          
+      - name: Pull srsim image
+        run: docker pull ghcr.io/srl-labs/containerlab/nokia_srsim:25.7.R1 && docker tag ghcr.io/srl-labs/containerlab/nokia_srsim:25.7.R1 nokia_srsim:25.7.R1
```

Comment:
let's retag to `registry.srlinux.dev/pub/nokia_srsim:25.7.R1` and use this in the test, since it will allow us to run the test locally as well.